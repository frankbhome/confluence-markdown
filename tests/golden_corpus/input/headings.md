# Level 1 Header

This is content under the main header.

## Level 2 Header

This is content under a sub-header.

### Level 3 Header

This is content under a sub-sub-header.
