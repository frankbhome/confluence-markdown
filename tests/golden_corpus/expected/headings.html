<h1>Level 1 Header</h1>

<p>This is content under the main header.</p>

<h2>Level 2 Header</h2>

<p>This is content under a sub-header.</p>

<h3>Level 3 Header</h3>

<p>This is content under a sub-sub-header.</p>
